Lib/test/test_ftplib.py

```diff
@@ -53,7 +55,7 @@ def __init__(self, conn, baseclass):
         self.baseclass.last_received_data = ''
 
     def handle_read(self):
-        self.baseclass.last_received_data += self.recv(1024).decode('ascii')
+        self.baseclass.last_received_data += self.recv(1024).decode('utf-8')
```

Comment:
Maybe add a self.encoding attribute to this class to avoid hardcoding the encoding?

Same for DummyFTPHandler.

Reply:
Thanks for the good review.

Just as the format of the TIMEOUT parameter, I have now instead added a single parameter ENCODING to the top of the file, which all the classes use. I checked that all tests also pass if using `latin-1` encoding, but I haven't added this test case, as I didn't want to more permanently lock future latin-1 support for both ftplib and all it's dependencies.

Reply:
Thanks, that's better.